### DISCUSSION TITLE
Add iframe on Documentation

### DISCUSSION THREAD

**Discussion by jonathan-arenas-edo** [mkdocs/mkdocs#3188] [Q&A]
Hello Guys,

Recently I'm working with Mkdocs with my Backstage and I have one doubt.
Is possible add one iframe on my README.md? I want to include a page link inside of my README.md

Thank you so much for your time.
Kind regards!

**Comment by pawamoy** [comment]
What have you tried? Did it work? If not, what errors did you get?

**Reply by jonathan-arenas-edo** [reply]
Hi @pawamoy ,

I tried to add an iframe but the page doesn't show nothing, it's empty.
I don't know if is it necessary a special format or what is the problem?

Example files:

> #README
> 
> This is test file and the iframe is:
> <iframe width="950" height="600" src="URL"></iframe>

**Comment by kamilkrzyskow** [comment]
@jarenasla-edo 
Hello, just wanting to make sure you're talking about displaying an `iframe` in your rendered MkDocs, not the README.md rendered on GitHub right?
The `<iframe>` tag should be normally supported (YouTube embeds work flawlessly), so if it's not working you'd have to look deeper, provide the exact/specific example here.
Open up your browser developer tools, you can see the HTML source to see if the iframe is in the DOM structure, and also you can check the console to see what is being loaded and if there is any error there.
